cura/Settings/MachineManager.py

```diff
@@ -220,15 +220,16 @@ def _onGlobalPropertyChanged(self, key, property_name):
                                 # Remove from the value from the other stacks as well, unless the
                                 # top value from the other stacklevels is different than the new value
                                 for container in extruder_stack.getContainers():
-                                    if container == extruder_stack.getTop():
-                                        continue
+                                    # Commented out this line for a bit. Don't know why it's here, but it is causing
+                                    # issues with reset button. 
```

Comment:
This was just to skip the top container in this check. The value is removed from the top profile if the first encountered non-top container contains the same value.